src/library-authoring/LibraryBlock/LibraryBlock.tsx

```diff
@@ -28,7 +28,9 @@ export const LibraryBlock = ({
   view,
 }: LibraryBlockProps) => {
   const iframeRef = useRef<HTMLIFrameElement>(null);
-  const [iFrameHeight, setIFrameHeight] = useState(50);
+  const defaultiFrameHeight = view === 'studio_view' ? 80 : 50;
```

Comment:
nit: I think this would be clearer if you moved the line from [below](https://github.com/openedx/frontend-app-authoring/pull/1649/files#diff-baaa8a52955d551edd8e8bc1ae1bdd6d96fa79dbce8b48a2a35ecfae1f4ca3b7L80) up here and used its value:

```suggestion
  const xblockView = view ?? 'student_view';
  const defaultiFrameHeight = xblockView === 'studio_view' ? 80 : 50;
```

Reply:
Fixed https://github.com/openedx/frontend-app-authoring/pull/1649/commits/bfc12d1fb802de0eead7f99a167d1fe78215a52c